feat(llc): allow setting http client adapter to use native http (#2005)
GetStream/stream-chat-flutter: packages/stream_chat/CHANGELOG.md

## unreleased

✅ Added

> - Add `httpClientAdapter` in order to use native http implementations

Reviewer: ```suggestion
- Added `httpClientAdapter` to `StreamHttpClient` that enables the use native http implementations
```
